onboard-app-guilherme/src/screens/login/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { Button, Text, TextInput, View } from "react-native";
+
+export function LoginScreen() {
+	return (
+		<View>
+			<Text>Bem-vindo(a) à Taqtile!</Text>
+
+			<TextInput placeholder="E-mail" />
+
+			<TextInput placeholder="Senha" secureTextEntry={true} />
```

Comment:
Nit: as props booleanas com valor true podem omitir o valor
```suggestion
			<TextInput placeholder="Senha" secureTextEntry />
```